db/src/main.js

```diff
@@ -1,6 +1,6 @@
 import util from 'util';
 import chalk from 'chalk';
-import { upgrade, downgrade } from 'taskcluster-db';
+import { upgrade, downgrade } from './upgrade.js';
```

Comment:
I'm not sure how this resolution works in the runtime code, but you can teach typescript about these types of things with the `paths` property in the tsconfig. I use it in my personal project for absolute paths, e.g. https://github.com/gregtatum/indie-web/blob/main/src/logic/app-logic.ts#L1

Reply:
yeah, I think this was done by mistake in the past. `package.json` uses [`link:` feature](https://github.com/taskcluster/taskcluster/blob/641db13f513c7a773bf342befdb670c5e8b803ca/package.json#L111)
here we don't even need any special named paths. 
Taskcluster services use this since all services and libraries were moved into a single monorepo many years ago to link common libraries that are also located inside same repo